chat-plugins/mafia.js

```diff
@@ -1542,19 +1597,20 @@ const commands = {
 			switch (args[0]) {
 			case 'forceadd':
 			case 'add':
+				if (!this.canTalk()) return;
 				let targetUser = Users(args[1]);
 				if ((!targetUser || !targetUser.connected) && args[0] !== 'forceadd') return this.errorReply(`User ${args[1]} not found. To forcefully add the user to the queue, use /mafia queue forceadd, ${args[1]}`);
 				if (hostQueue.includes(args[1])) return this.errorReply(`User ${args[1]} is already on the host queue.`);
 				hostQueue.push(args[1]);
-				this.sendReply(`User ${args[1]} has been added to the host queue.`);
+				room.add(`User ${args[1]} has been added to the host queue.`).update();
```

Comment:
If we are broadcasting who is getting added in the host queue, make sure to announce who added them in the queue too I think so.